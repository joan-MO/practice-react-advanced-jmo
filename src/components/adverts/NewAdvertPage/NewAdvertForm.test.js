import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import NewAdvertForm from './NewAdvertForm';

configure({adapter: new Adapter()});

describe('NewAdvertForm', () => {
  const props = {
    onSubmit: jest.fn(),
  };

  const render = () => shallow(<NewAdvertForm {...props} />);
  
  test('snapshot testing', () => {
    const wrapper = render();
    expect(wrapper).toMatchSnapshot();
  });

});
