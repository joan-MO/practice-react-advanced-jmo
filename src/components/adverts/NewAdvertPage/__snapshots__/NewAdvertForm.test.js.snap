// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewAdvertForm snapshot testing 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    name="name"
    onChange={[Function]}
    value=""
  />
  <input
    checked={true}
    name="sale"
    onChange={[Function]}
    type="checkbox"
  />
  <input
    name="price"
    onChange={[Function]}
    type="number"
    value={0}
  />
  <SelectTags
    multiple={true}
    name="tags"
    onChange={[Function]}
    value={Array []}
  />
  <InputFile
    name="photo"
    onChange={[Function]}
  />
  <button
    disabled={true}
  >
    Save
  </button>
</form>
`;
